### PR TITLE
[FLINK-32767][doc] Fix en-doc SHOW CREATE TABLE usage

### PR DESCRIPTION
**Pull request by GOODBOY008**
Fix en-doc SHOW CREATE TABLE usage.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@leonardBang Pls cc.

File: docs/content/docs/dev/table/sql/show.md [hunk 1]
```diff
@@ -608,7 +608,7 @@ show tables;
 ## SHOW CREATE TABLE
 
 ```sql
-SHOW CREATE TABLE
+SHOW CREATE TABLE [catalog_name.][db_name.]table_name
```
Comment (RocMarshal) [inline docs/content/docs/dev/table/sql/show.md:611]:
```suggestion
SHOW CREATE TABLE [[catalog_name.]db_name.]table_name
```

**Review by RocMarshal — COMMENTED** [review]
Thanks @GOODBOY008 for your contribution.
I left a minor comment, PTAL ~

File: docs/content/docs/dev/table/sql/show.md (hunk 1 above)
Comment (GOODBOY008) [reply docs/content/docs/dev/table/sql/show.md:611]: Most doc is  `[catalog_name.][db_name.]table_name` . @RocMarshal

Comment (leonardBang) [reply docs/content/docs/dev/table/sql/show.md:611]: https://github.com/apache/flink/blob/8c9b1767e08e3fdb3b60a0cc7f09a92647181d64/flink-table/flink-sql-client/src/test/resources/sql/catalog_database.q#L311

**Comment by RocMarshal** [comment]
Thanks @GOODBOY008 for the update & @leonardBang .
Maybe we need change the JIRA ticket title & the PR title to `Fix table identifier format in docs` or any-titles proper.
Because the rest of table identifier reference was updated.

**Comment by GOODBOY008** [comment]
@leonardBang @RocMarshal  I found sqlserver offical doc use this.
企业微信截图_16914603187821

**Comment by leonardBang** [comment]
> @leonardBang @RocMarshal I found sqlserver offical doc use this. Although the explanation is very redundant, it is very clear. 企业微信截图_16914603187821

+1, the sqlserver way is more friendly to users.

**Comment by GOODBOY008** [comment]
@leonardBang @RocMarshal In this pr, just modify `show create table` use `[[catalog_name.]db_name.]table_name`. Please code review.

**Review by RocMarshal — APPROVED** [review]
Thanks @GOODBOY008 for the update.
LGTM +1.
CC @leonardBang

**Review by leonardBang — APPROVED** [review]
Thanks @GOODBOY008 and @RocMarshal for the contribution, LGTM

**Pull request merged**